action/search.php

```diff
@@ -41,6 +41,7 @@ public function handle_indexing(Doku_Event $event, $param) {
         if(!$tables) return;
 
         foreach($tables as $table) {
+            // FIXME is timestamp relevant here?
```

Comment:
It should always be the newest available data be indexed. I believe a timestamp of zero used to load the newest available page data. If that's still the case this should be fine.

Reply:
Hm, this is of course no longer the case, so I changed the lines and now we explicitly pass the current timestamp to the query.